crowdin/config.yml

```diff
@@ -0,0 +1,29 @@
+#
+# Your crowdin's credentials
+#
+"api_key_env": CROWDIN_API_KEY
+"project_identifier_env": CROWDIN_PROJECT_ID
+"base_path_env": CROWDIN_BASE_PATH
+#"base_url" : ""
```

Comment:
dead code?

Reply:
Yes, this was generates by `crowdin-cli` itself.